docs/payment-gateways.rst

```diff
@@ -3,15 +3,16 @@
 Supported Payment Gateways
 ==========================
 
-Here is our list of supported payment gateways:
+Here is our list of supported payment gateways, and their configuration:
 
 .. toctree::
    :maxdepth: 1
 
    payment-gateways/braintree.rst
    payment-gateways/razorpay.rst
 
-The default configuration uses the `dummy` backend.
+The default configuration only uses the `dummy` backend
+(see :ref:`how to enable/ disable payment gateways <payment_gateways_configuration>`).
```

Comment:
Maybe enable/disable ?

Reply:
I just checked, the French rule doesn't apply here in this context, thanks :wink: